SwanCustomEnvironments/swancustomenvironments/scripts/makenv.sh

```diff
@@ -157,10 +162,17 @@ if [ -n "${CURRENT_ENV_NAME}" ]; then
     exit 1
 fi
 
-# Create environment (acc-py or generic)
+# Create environment (acc-py, conda or generic as default)
 if [ -n "$ACCPY_VERSION" ]; then
-    source $ACCPY_PATH/base/${ACCPY_VERSION}/setup.sh
-    acc-py venv ${ENV_PATH} | tee -a "${LOG_FILE}"
+    source "${ACCPY_PATH}/base/${ACCPY_VERSION}/setup.sh"
+    acc-py venv ${ENV_PATH} | tee -a ${LOG_FILE}
+elif [ -n "$CONDA_ENV" ]; then
+    _log "Creating environment ${ENV_NAME} using Mamba Conda with requirements from (${REQ_PATH})..."
```

Comment:
I would also change the message here to `Creating conda environment ${ENV_NAME}..." to make it consistent with the message below.

Reply:
It has not any message because the acc-py does that already:
```
[1;32m=> [0m Acc-Py base 2023.06.2 is now active within this shell.
INFO:acc_py_devtools:Creating virtual environment in /home/rhenriqu/std_python_env
Setting up the environment...
```

Reply:
Fair enough. But if you adopt the strategy of the auxiliary scripts, what I would do is just have one line:
```bash
_log "Creating environment ${ENV_NAME} using ${builder_name}..."
```
and after
```bash
source create_${builder_name}.sh
```